repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -51,18 +60,24 @@ class Interpreter(prompt0: Ref[String],
   val eval = Evaluator(mainThread.getContextClassLoader, 0  )
 
   val dynamicClasspath = new VirtualDirectory("(memory)", None)
-  var compiler: Compiler = _
+  var compiler: Compiler = null
   var pressy: Pressy = _
   def evalClassloader = eval.sess.frames.head.classloader
+
+  def updateClassPath = {
```

Comment:
Why does this have to be a global `var`? What's wrong with using `eval.sess.frames.head.classpath` directly?